apps/babel.config.json

```diff
@@ -9,8 +9,7 @@
     "@babel/plugin-proposal-object-rest-spread",
     "@babel/plugin-transform-class-properties",
     ["@babel/plugin-transform-regenerator", {"async": true, "asyncGenerators": true}],
-    // needed for IE 9/10 support
-    ["@babel/plugin-transform-classes", {"loose": true}],
+    "@babel/plugin-transform-classes",
```

Comment:
Informational comment: This should be included in preset-env and not necessary to be listed. However, to limit scope I think we can leave it and let the polyfill clean up work clean up the entire file.